src/contentscript/index.ts

```diff
@@ -21,6 +21,11 @@ import { OverlayManager } from './overlay/root/overlayManager'
 
 const IS_OVERLAY_IFRAME = window.name.indexOf('dapplet-overlay') !== -1
 const IS_E2E_ENV = isE2ETestingEnvironment(window)
+let injector: Injector
+
+export function getRegistriesInfo() {
```

Comment:
Circular dependency between `src/contentscript/index.ts` and `src/contentscript/overlay/root/components/Dapplet/index.tsx` increases code cohesion.